docs/project/changelog.md

```diff
@@ -287,7 +287,7 @@ _January 3, 2023_
 
 - {{ Enhancement }} Added a system for making Pyodide virtual environments. This
   is for testing out of tree builds. For more information, see [the
-  documentation](https://pyodide.org/en/stable/development/out-of-tree.html).
+  documentation](https://pyodide.org/en/stable/development/building-and-testing-packages.html).
```

Comment:
Thanks @JeffersGlass ! 

Actually, let's use the corresponding [sphinx reference](https://github.com/pyodide/pyodide/blame/main/docs/development/building-and-testing-packages.md#L1) (cf [MyST docs](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#markdown-links-and-referencing)) which should then auto-resolve to the correct URL (and will remain valid even once the version is archived later under a different base URL than /en/stable/)

```suggestion
  documentation](building-and-testing-packages-out-of-tree).
```

Also, you can add `[skip ci]` in the commit message, to skip all CI except for documentation.